docs/src/main/sphinx/release/release-418.md

```diff
@@ -0,0 +1,69 @@
+# Release 418 (17 May 2023)
+
+## General
+
+* Add support for `EXECUTE IMMEDIATE`. ({issue}`17341`)
+* Fix failure when invoking `current_timestamp`. ({issue}`17455`)
+
+## BigQuery connector
+
+* Add support for adding labels to BigQuery jobs started by Trino as part of
+  query processing. The name and value of the label can be configured via the
```

Comment:
can we just link to the docs for this instead of adding the doc inline in the release notes?

Reply:
Sorry .. wrong one..